compiler/optimizer/GlobalRegisterAllocator.cpp

```diff
@@ -2952,42 +2672,77 @@ void TR_GlobalRegisterAllocator::offerAllAutosAndRegisterParmAsCandidates(TR::Bl
       interestedBlocks.set(block->getNumber());
       }
 
+   // This first part isn't really looking for If-Then-Else candidates.
```

Comment:
I think we should delete this line 

```// This first part isn't really looking for If-Then-Else candidates.```

since it does not really belong anymore.

Reply:
Done.